msteams-platform/graph-api/meeting-transcripts/fetch-id.md

```diff
@@ -532,3 +532,4 @@ You can try the following code sample for a bot app:
 
 - [Apps for Teams meetings and calls](../../apps-in-teams-meetings/teams-apps-in-meetings.md)
 - [API reference for the Bot Framework Connector service](/azure/bot-service/rest-api/bot-framework-rest-connector-api-reference)
+- [Get notifications for Teams meeting call updates](/graph/changenotifications-for-onlinemeeting)
```

Comment:
`Get notifications for call updates` is not the question that the user raised. 
The user asked `How can I get the live participant data...`

Trying to base the doc update on to article title where the answer lies is a poor attempt for SEO. Rather mention what the user wanted instead of what the help article contains.